README.md

```diff
@@ -6,17 +6,77 @@
 
 ## Março
 
+### [Campus Party Amazônia](https://brasil.campus-party.org/campus-party-amazonia-2020/) (2020-03-18)
+* **Data:** 18/03/2020 a 22/03/2020
+* **Local:** Arena da Amazônia - Manaus
+* **Tags:** tecnologia, games, startups, inovação
+
+### [BrazilJS Fortaleza](https://braziljs.org/eventos/otr-fortaleza/2020) (2020-03-21)
+* **Data:** 21/03/2020
+* **Local:** Centro Universitário 7 de Setembro - Fortaleza-CE
+* **Tags:** javascript, node, web
+
 ## Abril
 
+### [TDC Belo Horizonte](https://thedevconf.com/tdc/2020/belohorizonte/trilhas) (2020-04-16)
+* **Data:** 16/04/2020 a 18/04/2020
+* **Local:** _locais a confirmar_
+* **Tags:** desenvolvimento, web, backend, python, node, ruby, java, agil, golang
+
 ## Maio
 
-#### [QCon São Paulo](https://qconsp.com) (2019-05-04)
-* **Data:** 04/05/2019 a 06/05/2019
+#### [Python Sul](https://pythonsul.org/) (2020-05-01)
+* **Data:** 01/05/2020 a 03/05/2020
+* **Local:** Centro de Inovação de Jaraguá do Sul - Jaraguá do Sul-SC
+* **Tags:** desenvolvimento, web, backend, fullstack, tecnologia
+
+#### [QCon São Paulo](https://qconsp.com) (2020-05-04)
+* **Data:** 04/05/2020 a 06/05/2020
 * **Local:** Centro de Convenções Frei Caneca - São Paulo-SP
 * **Tags:** desenvolvimento, web, backend, fullstack, tecnologia
 
+#### [UXConf](https://www.uxconf.com.br/) (2020-05-15)
+* **Data:** 15/05/2020 a 16/05/2020
+* **Local:** PUCRS - Porto Alegre-RS
+* **Tags:** ux
+
+#### [CapiConf](https://capiconf.com/2020/start) (2020-05-15)
```

Comment:
There are differences in the heading level, maybe we should normalize everything.